crates/ty_completion_eval/truth/internal-typeshed-hidden/main.py

```diff
@@ -0,0 +1,9 @@
+# This is a case where a symbol from an internal module appears
+# before the desired symbol from `typing`.
+#
+# We use a slightly different example than the one reported in
+# the issue to capture the deficiency via ranking. That is,
+# in ty#1274, the (current) top suggestion is the correct one.
+#
+# ref: https://github.com/astral-sh/ty/issues/1274#issuecomment-3345923575
+AwaiGen<CURSOR: typing.AwaitableGenerator>
```

Comment:
`typing.AwaitableGenerator` is a bit of a weird one because it doesn't actually exist at runtime -- and it's decorated with `@type_check_only`, so we should actually suppress it from autocompletions altogether (https://github.com/astral-sh/ty/issues/816).

`NoneType` might be quite a good one here? It exists in `_typeshed/__init__.pyi` and `types.pyi`, but (assuming you're using Python 3.10+), you'd never want it to be imported from the `_typeshed` module (which doesn't exist at runtime). You'd always want it imported from `types` (which does).

Reply:
Yes, perfect! Thank you for catching this.